jetty-jmx/pom.xml

```diff
@@ -42,6 +42,10 @@
       <artifactId>jetty-util</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
```

Comment:
I'm fine this the change above, but below `jetty-slf4j-impl` is both `provided` and `optional`.
I think it should be like elsewhere, in scope `test` if we use it in tests, or otherwise not even declared.

Reply:
Fixed